resources/views/admin/poeticType/create.blade.php

```diff
@@ -0,0 +1,89 @@
+@extends('admin.layouts.master')
+
+@section('title')
+   Add New PoeticType - Dashboard
+@endsection
+
+@section('css')
+    <link href="{{asset('assetsAdmin/bootstrap/css/bootstrap.min.css')}}" rel="stylesheet" type="text/css" />
+    <link href="{{asset('assetsAdmin/assets/css/plugins.css')}}" rel="stylesheet" type="text/css" />
+    <link href="{{asset('assetsAdmin/assets/css/scrollspyNav.css')}}" rel="stylesheet" type="text/css" />
+    <link href="{{asset('assetsAdmin/plugins/file-upload/file-upload-with-preview.min.css')}}" rel="stylesheet" type="text/css" />
+    <!---BEGIN CUSTOM STYLE FILE--->
+    <link href="{{asset('assetsAdmin/assets/css/scrollspyNav.css')}}" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" type="text/css" href="{{asset('assetsAdmin/plugins/select2/select2.min.css')}}">
+    <!--  END CUSTOM STYLE FILE  -->
+@endsection
+
+@section('content')
+    <div id="content" class="main-content">
+        <div class="container">
+            <div class="container">
+
+                <div class="page-header">
+                    <nav class="breadcrumb-one" aria-label="breadcrumb">
+                        <ol class="breadcrumb">
+                            <li class="breadcrumb-item"><a href="{{route('admin.dashboard')}}">Home</a></li>
+                            <li class="breadcrumb-item"><a href="{{route('admin.poeticType.index')}}">PoeticTypes</a></li>&nbsp;
+                            <li class="breadcrumb-item active" aria-current="page"><a href="{{URL::current()}}">Add New PoeticType</a></li>
+                        </ol>
+                    </nav>
+
+                </div>
+
+                <div class="container">
+                    <div class="row my-5 mx-auto">
+                        <div class="col-lg-12 layout-spacing">
+                            <form action="{{route('admin.poeticType.store')}}" method="post">
```

Comment:
the same form used in create file and edit file, so we need to create shard file called "form" and include form file in create and edit

Reply:
done